### ISSUE TITLE
Unretire! on a fortress fully on mountain tiles with embark anywhere enabled attempts to embark instead

### ISSUE THREAD

**Issue by Tachytaenius** [DFHack/dfhack#1987]
Well, yeah, just that :>

**Comment by lethosor** [comment]
What exactly is the difference?

**Comment by wolfboyft** [comment]
Unretire, the thing that the key labelled "Unretire!" should do: "How have your sturdy charges fared in your absence?", resumes the game
vs
Embark: Choose what items and units, group name and symbol etc to bring to the new embark

**Comment by wolfboyft** [comment]
Is that what you meant?

**Comment by lethosor** [comment]
Yes, thanks. Unfortunately it is difficult to determine whether DF _can_ embark in a certain position, so embark-anywhere just forcibly opens an embark dialog. I'd recommend disabling it for now when attempting to unretire.
